specification/src/main/asciidoc/license-efsl.adoc

```diff
@@ -53,9 +53,9 @@ specification is expressly prohibited.
 
 The notice is:
 
-"Copyright (c) 2018,2020 Eclipse Foundation. This software or
-document includes material copied from or derived from 
-Jakarta EE Platform Specification, https://jakarta.ee/specifications/platform/9/[]."
+"Copyright (c) 2018, 2020 Eclipse Foundation. This software or
```

Comment:
The copyright years should be replaced with [$date-of-document] placeholder to match the original text.
```suggestion
"Copyright (c) [$date-of-document] Eclipse Foundation. This software or
```

Reply:
Nice catch, @hussainnm!  Thanks.